### PR TITLE
Make the API instalable

### PR DESCRIPTION
**Pull request by moacirmoda**
Hey man, I just made some adjusts to make the API instalable and easy to update, when is pluged to another projects.

### CODE REVIEW

**Comment by LevPasha** [comment]
This branch has conflicts that must be resolved...

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from setuptools import setup
+
+setup(name='instagram-api',
+      version='0.1',
+      description='Unofficial instagram API, give you access to ALL instagram features (like, follow, upload photo and video and etc)! Write on python.',
+      url='https://github.com/moacirmoda/pygram',
```
Comment (ahmdrz) [inline setup.py]: be careful about this one

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      author='Lions Crayons',
```
Comment (ahmdrz) [inline setup.py]: and ...

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+      author_email='info@lionscrayons.com',
```
Comment (ahmdrz) [inline setup.py]: and ...

**Comment by moacirmoda** [comment]
man sorry... i've just copy the setup from another project and forgot to change the url. I will do a new pull request.

**Pull request closed without merging**